orchestration/src/main/java/com/sap/ai/sdk/orchestration/spring/OrchestrationChatOptions.java

```diff
@@ -21,12 +27,19 @@
 import lombok.val;
 import org.springframework.ai.chat.prompt.ChatOptions;
 
-/** Configuration to be used for orchestration requests. */
+/**
+ * Configuration to be used for orchestration requests.
+ *
+ * @since 1.2.0
+ */
+@Beta
```

Comment:
(Major)

Why are you destabilizing public API?
We can only allow that when PO gave approval.

Reply:
Sorry made a mistake